### ISSUE TITLE
Application Role not Supported on iOS and Android

### ISSUE THREAD

**Issue by StevenMilne-Sonocent** [tldraw/tldraw#8849]
The TLDraw canvas uses `role="application"` but this role [isn't well supported by mobile screen readers](https://cerovac.com/a11y/2024/03/aria-roleapplication-and-mobile-screen-readers/) such as VoiceOver on iOS. As a result the canvas isn't announced at all on iOS which is a failure of [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) since a blind person will not even be aware of it's presence.

The `role` attribute can take fallback roles like `role="application img"` which could help here, however when it falls back to the `img` role then none of the descendants are exposed to assistive tech (all children of `role="img"` are [assumed](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/img_role#all_descendants_are_presentational) to be `role="presentation"`). This could cause issues for any toolbars etc shown inside so I'm not sure what the best fix would be without moving that UI out of the application role area 🤔 

If UI is to stay within the `role="application"` element then it should probably get marked as `role="document"` as outlined in the [MDN article](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/application_role#description).

**Comment by mimecuvalo** [comment]
Thanks for flagging, we'll take a look.
